test/mocha/spidermonkey.js

```diff
@@ -10,7 +10,7 @@ describe("spidermonkey export/import sanity test", function() {
         var command = uglifyjs + " --self -cm --wrap SpiderUglify -o spidermonkey | " +
             uglifyjs + " -p spidermonkey -cm";
 
-        exec(command, function(err, stdout) {
+        exec(command, { maxBuffer: 1048576 }, function(err, stdout) {
```

Comment:
Fun fact: with the addition of [this line](https://github.com/mishoo/UglifyJS2/pull/2524/files#diff-230c53e13e3ee367aec7d49470eb42caR332), uglified `uglify-es` now exceeds [200KB](https://nodejs.org/dist/latest-v9.x/docs/api/child_process.html#child_process_child_process_exec_command_options_callback)

Reply:
Strange setting. Must be for out of control servers.

I see you've been busy with PRs today - nice work!